### PR TITLE
Add local endpoint & entities

### PR DESCRIPTION
**Pull request by SwiftyAlex**
Adds the data for the local endpoint. It doesn't add a CMS for it, but given this is unlikely to change im ok with that one personally 😄 

Should unblock local page being worked on 🙏🏻

Sample response:

```swift
{
    "data": [
        {
            "symbolName": "wineglass.fill",
            "id": "BD99AE67-B891-4EF0-B22A-81F89926555B",
            "name": "Wine",
            "locations": [
                {
                    "id": "BD99AE67-B891-4EF0-B22A-81F89926555B",
                    "lat": 53.796072000000002,
                    "long": -1.544208,
                    "name": "Alchemist",
                    "url": "https://thealchemist.uk.com/venues/leeds-trinity/"
                }
            ]
        }
    ]
}
```

Really simple response, just enough to show litle pins with URLs. Allows for design freedom too as you can re-use the category symbols.

### CODE REVIEW

File: Sources/App/Features/Activities/Controllers/ActivityAPIController.swift [hunk 1]
```diff
@@ -29,7 +29,7 @@ struct ActivityAPIController: RouteCollection {
         }
 
         // Image upload
-        var fileName = "\(UUID.generateRandom().uuidString)-\(image.activityImage.filename)"
+        let fileName = "\(UUID.generateRandom().uuidString)-\(image.activityImage.filename)"
```
Comment (SwiftyAlex) [inline Sources/App/Features/Activities/Controllers/ActivityAPIController.swift:32]: I know this change was a bit of a drive by but the warning was super annoying, my bad!

File: Sources/App/Features/Locations/Migrations/Location+Migration+V1.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import Vapor
+
+class LocationMigrationV1: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        try await database.schema(Schema.location)
+             .id()
+             .field("name", .string, .required)
+             .field("url", .string, .required)
+             .field("lat", .double, .required)
+             .field("long", .double, .required)
```
Comment (adamrushy) [inline Sources/App/Features/Locations/Migrations/Location+Migration+V1.swift:12]: It's a nitpick but should we name this "lon"

**Review by adamrushy — APPROVED** [review]
Small comment otherwise great work 😍

File: Sources/App/Features/Locations/Migrations/Location+Migration+V1.swift (hunk 2 above)
Comment (SwiftyAlex) [reply Sources/App/Features/Locations/Migrations/Location+Migration+V1.swift:12]: I never actually know which is correct when they're shorter 😅 I'll whack up a change

**Pull request merged**